public/fonts/inter/inter.css

```diff
@@ -196,5 +193,5 @@ explicitly, e.g.
   font-weight: 100 900;
   font-display: swap;
   font-style: oblique 0deg 10deg;
-  src: url("Inter.var.woff2?v=3.19") format("woff2");
+  src: url('Inter.var.woff2?v=3.19') format('woff2');
 }
```

Comment:
Please remove these linted `.md` and `.css` files from this pull request.  We will have our own processes for getting these linted, and these are not relevant to the change being implemented.

Reply:
TY! 🙏